src/query/intersection.rs

```diff
@@ -11,14 +12,14 @@ use crate::{DocId, Score};
 /// For better performance, the function uses a
 /// specialized implementation if the two
 /// shortest scorers are `TermScorer`s.
-pub fn intersect_scorers(mut scorers: Vec<Box<dyn Scorer>>) -> Box<dyn Scorer> {
+pub fn intersect_scorers(mut scorers: Vec<Box<dyn Scorer>>, num_docs: u32) -> Box<dyn Scorer> {
```

Comment:
What is num_docs here?
This is unclear.

Is it an estimation of the number of docs in the intersection? max_doc?

Reply:
number of docs in the segment, renamed and added a comment